### ISSUE TITLE
copy(\\ASUS-LAPTOP-AQSA\BlackCopper 80mm Series): Failed to open stream: No such file or directory

### ISSUE THREAD

**Issue by hamza-saqib** [charlieuki/receipt-printer#30]
cant print, i have set the exact name of my printer, its thermal printer from BlackCopper, Moel -> 'BlackCopper 8mm Series'. i am working on Laravel 8

**Comment by FSHK** [comment]
Bruh... could you solve it? I had the same issue you just need to enable sharing on the printer properties as this guy said here  : [here](https://github.com/charlieuki/receipt-printer/issues/18), did it on my laptop with a BEMATECH LR2000E

**Comment by hamza-saqib** [comment]
yeah i have figured it out. and was working fine. but somehow i don't know but enabling that sharing option isn't working now on my system.
